Makefile

```diff
@@ -1,7 +1,9 @@
 all: install
 install: clean
+	#on python 2.6, pre-install last supported version of numpy
+	python --version 2>&1 >/dev/null | grep '2.6' && pip install 'numpy==1.10.0' || true
 	python setup.py sdist
-	pip install dist/*.tar.gz --upgrade
+	pip install dist/*.tar.gz --upgrade --upgrade-strategy=only-if-needed
```

Comment:
Right now it's using numpy's that are installed in the travis system, and not upgrading them; which makes for a faster build, but we aren't guaranteed to get the latest.  I can make this conditional, too, if that makes you more comfortable with it.